### DISCUSSION TITLE
Need Any Help?

### DISCUSSION THREAD

**Discussion by benfl3713** [benfl3713/DepartureBoard#176] [Q&A]
Ask any questions you may have about Led Departure Board and we'll try get back to you as soon as possible

**Comment by KyeTheHuskyPup** [comment]
hey there when would there be an tram feature

**Reply by ben-fletcher** [reply]
Hi @KyeTheHuskyPup Is there a particular **city/region** you have in mind, as there is no general uk datasouce?

**Reply by KyeTheHuskyPup** [reply]
hey just seen this , west midlands if possible please

**Reply by ben-fletcher** [reply]
Good choice, I believe they're 1 of the few that can provide this data. I'll do some investigation into how to use their data 👍

**Reply by KyeTheHuskyPup** [reply]
hey this may help https://api-portal.tfwm.org.uk/ 👍

**Comment by Beenster** [comment]
Hi, hoping you see this a few years later - this is an awesome thing. I’m getting a “black screen” on an Android wall panel (Sonoff NS Panel Pro) device (not any other device) which I suspect is due to the absence of font available on device or maybe ability to download the font. Do you have any tips for remediation or configuration options? Many thanks…

**Reply by benfl3713** [reply]
Hi @Beenster the custom fonts used for the site is actually downloaded from my site rather than using one from the device, so this should not be the issue. Typically a black page (even when trying to load the home page) normally indicates that the website is not able to run javascript. To which my entire website relies on javascript to display anything.

My suggestions would be to ensure you haven't disabled javascript from running (You should know if you have done this as by default, browsers are allowed to use javascript). Secondly possibly try a different browser if possible, could be the one you're using is outdated. And thirdly if you try and use google chrome and it still doesn't work, you could try using [chrome remote devtools](https://developer.chrome.com/docs/devtools/remote-debugging) to see what the logs are which should highlight what is failing to start correctly.

Happy to help out with the devtools if needed

**Comment by mlinds1989** [comment]
Hey - So this is a mega cool website and I've been trying to display our pubs train times on a screen for forever and found you. So it's absolutely fantastic and I've got the LHA station showing on a tablet. 
Question is we now have Yodeck as a digital signage tool and I'm desperately trying to embed the website using their UI, I don't really know what I'm doing! HA - the page shows the cookie screen - and I've tried using the "?acceptCookies=Yes) URL but it shall not work! 
It looks like it's just continually reloading. 
Can you help? TYVMIA

**Reply by benfl3713** [reply]
Hi thre @mlinds1989 Great to hear your trying to use this in a pub!
I think you've found a little problem with the way that the acceptCookies url works. It's accepts the cookies but has to reload the page afterwards, however a lot of digital signage systems reset the browser on reload so it forgets that you set the cookies.

I shall look into this this evening as I think I can change how the acceptCookies flag works to avoid a page reload. I'll keep you posted on what I achieve

**Reply by benfl3713** [reply]
I've taken a look at this and have changed how the code works so that it doesn't reload the page. You should be able to use it without the cookie banner now. Just a note, rather than passing in "Yes" you should use "true" or "false".
E.g `?acceptCookies=true`

**Comment by mlinds1989** [comment]
You Ben, are a legend.

I know of quite a few businesses that could benefit from your site. I know
you have a "buy me a coffee" so I will in due course. You should perhaps
add a £9.99 or something link on your site for business users to sign up,
maybe they could add their business name to the top line in return?

I'm very happy to promote the site and will link in with our digital
displays. If you're ever near Loch Awe. Dinners on me!

Mike - Ben Cruachan Inn.

**Comment by pruppertjr** [comment]
For Tube timings, I put in my local station but I only get results for one direction, is there any way to get the other or both directions at once?
